src/consensus/aft/raft_types.h

```diff
@@ -121,12 +134,26 @@ namespace aft
     Term term_of_idx;
     bool contains_new_view;
   };
+  DECLARE_JSON_TYPE_WITH_2BASES(
+    AppendEntries, RaftHeader, consensus::AppendEntriesIndex);
+  DECLARE_JSON_REQUIRED_FIELDS(
+    AppendEntries,
+    term,
+    prev_term,
+    leader_commit_idx,
+    term_of_idx,
+    contains_new_view);
```

Comment:
do we want to change to either view or term throughout?

Reply:
Moved to #5213